### ISSUE TITLE
Add a type for apomorphies in TaxonomicUnitWrapper

### ISSUE THREAD

**Issue by gaurav** [phyloref/phyx.js#28]
TaxonomicUnitWrapper has a TYPE_TAXON_CONCEPT and a TYPE_SPECIMEN. We should add a TYPE_APOMORPHY for specifiers that are apomorphies.

**Comment by hlapp** [comment]
I agree http://purl.obolibrary.org/obo/CDAO_0000071 is a good candidate. It's certainly not wrong, and I'm not sure we need a more specific type than that.

**Comment by hlapp** [comment]
> I agree http://purl.obolibrary.org/obo/CDAO_0000071 is a good candidate.

I'm not so sure anymore, actually. An apomorphy is really a character state, a phenotype, not a character. Maybe use something neutral, such as <https://semanticscience.org/resource/SIO_010056>.
